src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -51,33 +79,36 @@ Ethereum does not support a large population of light nodes yet, but light node
 
 ### Archive node {#archive-node}
 
-- Stores everything kept in the full node and builds an archive of historical states. Needed if you want to query something like an account balance at block #4,000,000, or simply and reliably [test your own transactions set without mining them using OpenEthereum](https://openethereum.github.io/JSONRPC-trace-module#trace_callmany).
-- These data represent units of terabytes which makes archive nodes less attractive for average users but can be handy for services like block explorers, wallet vendors, and chain analytics.
+- Stores everything kept in the full node and builds an archive of historical states. It is needed if you want to query something like an account balance at block #4,000,000, or simply and reliably test your own transactions set without mining them using tracing. 
+- This data represents units of terabytes, which makes archive nodes less attractive for average users but can be handy for services like block explorers, wallet vendors, and chain analytics.
 
-Syncing clients in any mode other than archive will result in pruned blockchain data. This means, there is no archive of all historical states but the full node is able to build them on demand.
+Syncing clients in any mode other than archive will result in pruned blockchain data. This means, there is no archive of all historical states but the full node is able to build them on demand. 
 
 ## Why should I run an Ethereum node? {#why-should-i-run-an-ethereum-node}
 
-Running a node allows you to trustlessly and privately use Ethereum while supporting the ecosystem.
+Running a node allows you to directly, trustlessly and privately use Ethereum while supporting the network by keeping it more robust and decentralized. 
 
 ### Benefits to you {#benefits-to-you}
 
 Running your own node enables you to use Ethereum in a truly private, self-sufficient and trustless manner. You don't need to trust the network because you can verify the data yourself with your client. "Don't trust, verify" is a popular blockchain mantra.
```

Comment:
I believe it is because no information about transactions or addresses is leaked into network. Only IP of the client is 'leaked' to peers but any private data. Later in the text it explains the scenario with a wallet connection to own RPC. It also says 'enables you to use..', it's not supposed to say it automatically does without any effort. Maybe I would specify it better in the initial text